nbgrader/apps/feedbackapp.py

```diff
@@ -37,38 +26,74 @@ class FeedbackApp(BaseNbConvertApp):
 
     examples = Unicode(dedent(
```

Comment:
Not a dumb question, that's actually a good question. I'm not sure. The `examples` are something special having to do with `Application` I think (they are displayed when you do `--help`), but I don't know if they actually need to be traitlets.